tests/formats/test_json.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import pytest
+import tornado
+
+from tchannel import TChannel, schemes
+from tchannel import response
+from tchannel.tornado import TChannel as DeprecatedTChannel
+
+
+@pytest.mark.gen_test
+@pytest.mark.call
+def test_call_should_get_response():
+
+    # Given this test server:
+
+    server = DeprecatedTChannel(name='server')
+
+    @server.register('endpoint', schemes.JSON)
+    @tornado.gen.coroutine
+    def endpoint(request, response, proxy):
+
+        header = yield request.get_header()
+        body = yield request.get_body()
+
+        assert header == {'req': 'header'}
+        assert body == {'req': 'body'}
+
+        response.write_header({
+            'resp': 'header'
+        })
+        response.write_body({
+            'resp': 'body'
+        })
+
+    server.listen()
+
+    # Make a call:
+
+    tchannel = TChannel(name='test')
+
+    resp = yield tchannel.json(
+        service=server.hostport,
+        endpoint='endpoint',
+        header={'req': 'header'},
```

Comment:
@blampe like here you're right, `headers` makes way more sense, unless we always want people to think of it as the `header` block... but that makes them have to care about the transport, probably more than makes sense...